backend/services/automations/src/executions/calculateExecutions.ts

```
@@ -0,0 +1,113 @@
+}: {
+  models: IModels;
+  subdomain: string;
+  automationId: string;
+  trigger: ITrigger;
+  target: any;
+}): Promise<IAutomationExecutionDocument | null | undefined> => {
+  const { id, type, config, isCustom } = trigger;
+  const { reEnrollment, reEnrollmentRules, contentId } = config || {};
+
+  try {
+    if (!!isCustom) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)